gcp/appengine/frontend_handlers.py

```diff
@@ -77,6 +79,54 @@ def index():
   return render_template('index.html')
 
 
+@blueprint.route('/v2/')
+def index_v2():
+  return render_template('home.html')
+
+
+@blueprint.route('/v2/list')
+def list():
+  """Main page."""
+  # TODO: Can/should we do this as an internal query not a full on request to osv.dev?
+  page = request.args.get('page') if request.args.get('page') and request.args.get('page').isnumeric() else '1'
+  response = requests.get('https://osv.dev/backend/query?page=%s&search=&affected_only=true&ecosystem=' % page)
```

Comment:
Instead of doing this, let's just factor out the querying part of `query_handler` in this file and call this here directly.

i.e. something similar to:

```python
def run_query(search_string, page, affected_only, ecosystem):
  query = osv.Bug.query(osv.Bug.status == osv.BugStatus.PROCESSED,                                                                                                                                                                                   
                        osv.Bug.public == True)  # pylint: disable=singleton-comparison)
   ...

@blueprint.route(_BACKEND_ROUTE + '/query')                                                                                                                                                                                                          
def query_handler():
  search_string = request.args.get('search')                                                                                                                                                                                                         
  page = int(request.args.get('page', 1))                                                                                                                                                                                                            
  affected_only = request.args.get('affected_only') == 'true'                                                                                                                                                                                        
  ecosystem = request.args.get('ecosystem')   
  return jsonify(run_query(search_string, page, affected_only, ecosystem))

@blueprint.route('/v2/list')
def list():
  results = run_query(...)
   ...
```

Reply:
Done.